README.md

```diff
@@ -0,0 +1,196 @@
+https://github.com/ethereum/remix-project/releases also gives others ways to use Remix locally. Please check it out.
+
+Remix consists of many modules and in this repository you will find the Remix IDE (aka. Browser-Solidity).
+
+![Remix screenshot](https://github.com/ethereum/remix-ide/raw/master/remix_screenshot.png)
+
+## Offline Usage
+
+The `gh-pages` branch has always the latest stable build of Remix. It also contains a ZIP file with the entire build. Download it to use offline.
+
+Note: It contains the latest release of Solidity available at the time of the packaging. No other compiler versions are supported.
+
+
+## INSTALLATION:
+
+Install **npm** and **node.js** (see https://docs.npmjs.com/getting-started/installing-node), then
+install [Nx CLI](https://nx.dev/react/cli/overview) globally to enable running **nx executable commands**.
+```bash
+npm install -g @nrwl/cli
+```
+
+clone the github repository (`wget` need to be installed first) :
+
+```bash
+git clone https://github.com/ethereum/remix-project.git
+
+cd remix-project
+npm install
+nx build remix-ide --with-deps
+nx serve
+```
+
+## Docker:
+
+Prerequisites: 
+* Docker (https://docs.docker.com/desktop/)
+* Docker-compose (https://docs.docker.com/compose/install/)
+
+### Run with docker
+
+If you want to run latest changes that are merged into master branch then run:
+
+```
+docker pull remixproject/remix-ide:latest
+docker run -p 8080:80 remixproject/remix-ide:latest
+```
+
+If you want to run latest remix-live release run.
+```
+docker pull remixproject/remix-ide:remix_live
+docker run -p 8080:80 remixproject/remix-ide:remix_live
+```
+
+### Run with docker-compose:
+
+To run locally without building you only need docker-compose.yaml file and you can run:
+
+```
+docker-compose pull
+docker-compose up -d
+```
+
+Then go to http://localhost:8080 and you can use you Remix instance.
+
+To fetch docker-compose file without cloning this repo run:
+```
+curl https://raw.githubusercontent.com/ethereum/remix-ide/master/docker-compose.yaml > docker-compose.yaml
+```
+
+## DEVELOPING:
+
+Run `nx serve` and open `http://127.0.0.1:8080` in your browser.
+
+Then open your `text editor` and start developing.
+The browser will automatically refresh when files are saved.
+
+### Troubleshooting building
+
+Some things to consider if you have trouble building the package:
+
+- Make sure that you have the correct version of `node`, `npm` and `nvm`. Also ensure you have [Nx CLI](https://nx.dev/react/cli/overview) installed globally. You can find the version that is tested on Travis CI by looking at the log in the [build results](https://travis-ci.org/ethereum/remix-ide).
+
+Run:
+
+```bash
+node --version
+npm --version
+nvm --version
+```
+
+- In Debian based OS such as Ubuntu 14.04LTS you may need to run `apt-get install build-essential`. After installing `build-essential` run `npm rebuild`.
+
+## Unit Testing
+
+Run the unit tests via: `nx test <project-name>`
+```bash
+    nx test remix-libs
```

Comment:
I think it is better to put a relevant command as we don't have a project with name `remix-libs`